common/model-views.cpp

```diff
@@ -1936,12 +1936,23 @@ namespace rs2
         {
             if (it->second)
             {
-                auto format = format_values.at(it->first)[ui.selected_format_id.at(it->first)];
-                if (format == RS2_FORMAT_Y16)
+                int selected_format_index = -1;
+                if (ui.selected_format_id.count(it->first) > 0)
+                    selected_format_index = ui.selected_format_id.at(it->first);
+
+                if (format_values.count(it->first) > 0 && selected_format_index > -1)
                 {
-                    is_cal_format = true;
-                    break;
-                }
+                    auto formats = format_values.at(it->first);
+                    if (formats.size() > selected_format_index + 1)
```

Comment:
So... if the index is `0` and the size is `1`, it won't go in?

Reply:
corrected